spec/rails_helper.rb

```diff
@@ -48,6 +48,13 @@ def visit_sandbox
   visit page_path(id: 'sandbox')
 end
 
+def click_download_panel
+  expect(page).to have_css('.sul-embed-download-panel', visible: false)
+  page.find('button[data-sul-embed-toggle="sul-embed-download-panel"]', visible: true)
+  page.find('[data-sul-embed-toggle="sul-embed-download-panel"]', match: :first).click
+  expect(page).to have_css('.sul-embed-download-panel', visible: true)
+end
```

Comment:
Totally good point.  My thinking here is, if I write a test that uses this helper (and am presumably making assertions about the content inside the download panel) I want my test to fail because of the expectations I make on the content.  If the test fails because the download panel failed to open/is not present, then the failure raised would point to the expectation in the helper instead of the expectation in my test.

I don't feel strongly about it either, though.  So totally willing to kill it.